x-pack/plugins/security/server/plugin.ts

```diff
@@ -295,12 +295,14 @@ export class SecurityPlugin
       spaces,
       audit: this.auditSetup,
       authz: this.authorizationSetup,
+      getCurrentUser: (request) => this.getAuthentication().getCurrentUser(request),
```

Comment:
__optional nit (only if you have other changes that will re-trigger CI, otherwise feel free to ignore):__ since we use the same arrow function in 4 places now, would you mind extracting it into a separate const to re-use it in all these 4 places?
```
    const getCurrentUser = (request: KibanaRequest) =>
      this.getAuthentication().getCurrentUser(request);
    this.auditSetup = this.auditService.setup({
      ...
      getCurrentUser,
      ...
    });
```

Reply:
Updated 👍

Reply:
Nice, thank you!